frontend/src/pages/meditation/components/meditation-timer/components/timer-button/timer-button.tsx

```diff
@@ -0,0 +1,50 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  isActive: boolean;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: string;
+  name: string;
+  duration: string;
+  unit: string;
+};
+
+const TimerButton: React.FC<Properties> = ({
+  isActive,
+  onChange,
+  value,
+  name,
+  duration,
+  unit,
+}) => {
+  return (
+    <label
+      className={getValidClassNames(
+        styles['timer-button'],
+        isActive && styles['active'],
+      )}
+    >
+      <input
+        type="radio"
```

Comment:
It's better to create separate common `Radio` component for such a purpose

Reply:
May be not now, but for the further project extensions it'll be good

Reply:
Let's move it to another task